src/jit/decomposelongs.cpp

```diff
@@ -1581,9 +1581,17 @@ GenTree* DecomposeLongs::DecomposeMul(LIR::Use& use)
     Range().Remove(op1);
     Range().Remove(op2);
 
+#if defined(_TARGET_X86_)
     tree->gtOp.gtOp1 = op1->gtGetOp1();
```

Comment:
Should update the header comment, since it says"edx:eax" -- say something about arm?